routeros/resource_ipv6_dhcp_client.go

```diff
@@ -0,0 +1,138 @@
+package routeros
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+// ResourceIPv6DhcpClient https://help.mikrotik.com/docs/display/ROS/DHCP#DHCP-DHCPv6Client
+func ResourceIPv6DhcpClient() *schema.Resource {
+	resSchema := map[string]*schema.Schema{
+		MetaResourcePath: PropResourcePath("/ipv6/dhcp-client"),
+		MetaId:           PropId(Id),
+
+		"add_default_route": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Default:     false,
+			Description: "Whether to add default IPv6 route after a client connects.",
+		},
+		"address": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "IPv6 address, which is assigned to DHCPv6 Client from the Server.",
+		},
+		KeyComment: PropCommentRw,
+		"default_route_distance": {
+			Type:         schema.TypeInt,
+			Optional:     true,
+			Default:      1,
+			Description:  "Distance of default route. Applicable if add-default-route is set to yes.",
+			ValidateFunc: validation.IntBetween(0, 255),
+			// Default route distance returns as empty when the dhcp-client is searching.
+			// This produces inconsistent results, for this case, we will suppress changes.
+			DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
I don't have an IPv6 network at hand, could you test this field with ```Default``` excluded and ```DiffSuppressFunc: AlwaysPresentNotUserProvided```? Will the desired schema behavior be present in this combination?

Reply:
Seems to work with your requested changes. I just used the same solution as already present in ResourceDhcpClient for /ip/dhcp-client.

Reply:
Now there is a small mix of different realizations of the same fucntional. Michael Dokolin suggested to use this variant for all parameters returning values, so that it would be easier to ensure compatibility of different versions of resource schemas in the future.

Reply:
I see, will adapt.